### DISCUSSION TITLE
github ID

### DISCUSSION THREAD

**Discussion by enoch-lightning** [1101-GS4719-C/Python-Community-GS4719-C#8] [Q&A] [answered]
想請問老師，老師在上課時，說要產生github ID。是要去哪裡產生呢？還是我本來就會有github ID呢？謝謝老師。

**Comment by chia-kai-chang** [comment, marked as answer]
@enoch-lightning 登入Github後，按下右上方的頭像按鈕，就可以看到Github ID。也就是Github的用戶名稱。

![image](https://user-images.githubusercontent.com/1768891/134107125-1b5d7f1f-48f1-424c-b185-02dc373bb306.png)

**Reply by enoch-lightning** [reply]
謝謝老師。看來老師對於使用Github真的很熟悉。

**Reply by chia-kai-chang** [reply]
答案正確記得
![image](https://user-images.githubusercontent.com/1768891/134211637-1d2dcb98-96ae-4250-ae62-f01517e9cc79.png)
唷!!
